Remotion/Web/Core/ResourceTheme.cs

```diff
@@ -30,6 +30,15 @@ public NovaGray ()
       }
     }
 
+    [ImplementationFor (typeof (ResourceTheme), Lifetime = LifetimeKind.Singleton)]
```

Comment:
```suggestion
```
There's only one registration intended. right now, we keep NovaGray, later on, this will be switched, when the new theme becomes the default.

please add an `[Obsolete ("'NovaViso' theme is work in progress and not intended for production use at this time.")]

Reply:
Makes sense because quite some tests failed for this reason - as I later found out :)